code/game/gamemodes/clock_cult/clock_structures/tinkerers_cache.dm

```diff
@@ -18,7 +18,9 @@
 	START_PROCESSING(SSobj, src)
 	clockwork_caches++
 	update_slab_info()
-	SetLuminosity(2,1)
+	#warn falloff?
```

Comment:
falloff was basically a cheap way to do light strength so just making this very weak light(which should be possible with this?), will be enough